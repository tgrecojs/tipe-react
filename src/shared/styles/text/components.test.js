import React from 'react'
import * as emotion from 'emotion'
import { createSerializer } from 'jest-emotion'
import styled from 'react-emotion'
import renderer from 'react-test-renderer'

expect.addSnapshotSerializer(createSerializer(emotion))

const Button = styled('div')`
  color: hotpink;
`
const H1 = styled.h1(
  {
    fontSize: 20
  },
  props => ({ 
    color: props.color,
    fontSize: props.fontSize
  })
)

test('H1', () => {
  expect(renderer.create(<H1></H1>))
})

test('H1', () => {
  expect(
    renderer.create(<H1 fontSize="24" color="yellow">Button</H1>).toJSON()
  ).toMatchSnapshot()
})