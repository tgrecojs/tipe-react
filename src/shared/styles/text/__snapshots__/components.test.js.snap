// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`H1 1`] = `
.emotion-0 {
  font-size: 20px;
  color: yellow;
}

<h1
  className="emotion-0 emotion-1"
  color="yellow"
>
  Button
</h1>
`;
